resources/cdk/redshift_item_tracker/README.md

```diff
@@ -0,0 +1,157 @@
+# Amazon Redshift item tracker sample application resources
+
+## Overview
+
+Creates AWS resources for Amazon Redshift sample applications. The scripts in this
+example create the following resources:
+ 
+* An AWS Secrets Manager secret that contains administrator credentials in a format 
+that can be used by an Amazon Redshift database. 
+* A Redshift cluster and database configured to use the credentials from the secret.
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Deploying resources
+
+You can use the AWS Cloud Development Kit (AWS CDK) or the AWS Command Line Interface
+(AWS CLI) to deploy and destroy the resources for this example.
+
+### Deploying with the AWS CDK
+
+To deploy with the AWS CDK, you must install [Node.js](https://nodejs.org) and the 
+[AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html).
+
+This example was built and tested with AWS CDK 2.33.0.
+
+Deploy AWS resources by running the following at a command prompt in the
+`resources/cdk/redshift_item_tracker` folder:
+
+```
+npm install
+cdk deploy
+```
+
+The stack takes a few minutes to deploy. When it completes, it prints output like 
+the following:
+
+```
+Outputs:
+RedshiftItemTrackerStack.ClusterId = doc-example-work-items-cluster
+RedshiftItemTrackerStack.Database = workitemtracker
+RedshiftItemTrackerStack.SecretArn = arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleredshiftworkitems-QlmFqEXAMPLE-nsoAF0
+```
+
+### Deploying with the AWS CLI 
+
+To deploy with the AWS CLI you must first install the 
```

Comment:
Insert comma after AWS CLI, like so: "To deploy with the AWS CLI, you must first..."

Reply:
fixed